assets/data/cards_pt-br.json

```
@@ -654,6 +663,15 @@
       ],
       "description": "Também chamados de modelos de referência, são modelos de aprendizado de máquina pré-treinados em grandes conjuntos de dados e comumente disponibilizados para uso por terceiros. Eles são usados como modelos de base para diversas aplicações de visão computacional, processamento de linguagem natural, entre outros. Modelos de base podem ser usados congelados, como uma camada de extração de características (features), ou passarem por um ajuste fino (finetune), gerando um modelo completo."
     },
+    {
+      "title": "Next.js",
+      "tags": [
+        "Conceito",
+        "Front-end",
+        "Framework"
+      ],
+      "description": "Next.js é um framework baseado em React que facilita a criação de sites com renderização no servidor (SSR) e geração de páginas estáticas (SSG), proporcionando melhor SEO e carregamento mais rápido de páginas."
```

Simplifiquei alguns termos técnicos e expliquei brevemente os benefícios do SSR e SSG. O que acha?

```suggestion
      "description": "Next.js é uma ferramenta baseada no React que ajuda a criar sites mais rápidos e fáceis de encontrar nos buscadores (SEO). Ele permite que as páginas sejam geradas no servidor (SSR) ou criadas de forma estática (SSG), o que melhora o tempo de carregamento das páginas e a experiência da pessoa usuária."
```